packages/diagrams/frontend/sirius-components-diagrams/src/renderer/edge/MultiLabelEdge.tsx

```diff
@@ -80,10 +73,19 @@ export const MultiLabelEdge = memo(
   }: MultiLabelEdgeProps<Edge<MultiLabelEdgeData>>) => {
     const { beginLabel, endLabel, label, faded } = data || {};
     const theme = useTheme();
-
+    const { zoom } = useViewport();
```

Comment:
Wouldn't this re-render all edges when the user will zoom on the viewport? Could you check this?

Reply:
Indeed, I get a re-rendering every time I change the zoom.